markdown/kubernetes_operator_codelab/kubernetes_operator_codelab.md

```diff
@@ -0,0 +1,243 @@
+```
+![Environment Setup Verification](img/Operator_Img2.png)
+
+### Next Step is to install the Operator
+
+
+## Install Solace Pubsub+ Event Broker Operator
+
+Duration: 0:05:00
+
+An operator is a containerized application that works with the Kubernetes Operator Framework to deploy, manage and monitor other containerized applications in a declarative way.  The PubSub+ Operator deploys the PubSub+ Event broker in stand-alone or high-availability configurations. The [User Guide](https://github.com/SolaceProducts/pubsubplus-kubernetes-operator/blob/main/docs/EventBrokerOperatorUserGuide.md) for the PubSub+ Operator describes all of its capabilities.
+
+You can install the Operator in two ways:
+
+#### Option A: Operator Hub and OLM install
+Follow the steps from [OperatorHub](https://operatorhub.io/operator/pubsubplus-eventbroker-operator) to first setup OLM, then to install the PubSub+ Event Broker Operator. Click on the Install button to see the detailed instructions.
+#### Option B: Direct Install
+Today, you will learn to do the direct install.
+
+```bash
+# clone the directory
+git clone git@github.com:SolaceProducts/pubsubplus-kubernetes-operator.git
+
+# cd to the pubsubplus-kubernetes-operator/deploy folder and apply deploy.yaml
+# Manifest creates a namespace and all K8s resources for the Operator deployment
+kubectl apply -f deploy.yaml
+
+# Wait for deployment to complete
+kubectl get pods -n pubsubplus-operator-system --watch
+```
+Note: If you do not have SSH setup on your machine and got an error cloning the repo, you can clone the https link instead `https://github.com/SolaceProducts/pubsubplus-kubernetes-operator.git`
+
+
+![Operator Setup Verification](img/Operator_Img3.png)
+
+### Next Step is to install the Event Broker
+
+## Install Solace Pubsub+ Event Broker 
+
+We are going to deploy a dev instance of the Solace Pubsub+ Event Broker using the Operator. Below steps are creating the “developer.yaml” that contains the configuration which the operator will deploy.  The format of the template is in [PubSub+ Operator API Parameters Reference](https://github.com/SolaceProducts/pubsubplus-kubernetes-operator/blob/main/docs/EventBrokerOperatorParametersReference.md) .  Then you are using kubectl to deploy the template.
+
+### Minimum-footprint Deployment for Developers
+```bash
+# This command returns the list of worker nodes and their status
+# Create deployment manifest
+
+echo "
+apiVersion: pubsubplus.solace.com/v1beta1
+kind: PubSubPlusEventBroker
+metadata:
+  name: dev-example
+spec:
+  developer: true" > developer.yaml
+
+# Then apply it
+kubectl apply -f developer.yaml
+
+# Wait for broker deployment pods to be ready
+kubectl get pods --show-labels --watch
+
+# Check service-ready
+kubectl wait --for=condition=ServiceReady eventbroker dev-example
+```
+
+![Broker Setup Verification](img/Operator_Img4.png)
+
+### Next Step is to test the deployment
+
+## Test Solace Pusbub+ Event Broker Deployment
+
+Duration: 0:05:00
```

Comment:
I believe this step is more than 5 minutes... can we increase this to 15 minutes